pkg/tfgen/generate_python.go

```diff
@@ -585,8 +578,8 @@ func (g *pythonGenerator) emitResourceType(mod *module, res *resourceType) (stri
 
 	// Override translate_{input|output}_property on each resource to translate between snake case and
 	// camel case when interacting with tfbridge.
-	w.Writefmtln(`
```

Comment:
Do we not need this newline in the output?

Reply:
we do not.  There is already a blank line above.  This leads to a slightly ungainly double blank line.

Reply:
It's also not a big deal.  the primary thing i wanted to improve was the awful assignment into props above.  This was just tiny cleanup along the way.